cvxpy/tests/test_KKT.py

```diff
@@ -183,13 +183,13 @@ class StandardTestFlags(BaseTest):
     def tf_1() -> STH.SolverTestHelper:
         """
         Tests NSD flag
+        Reference values via SCS
```

Comment:
The default termination criteria for SCS are very loose, and they can lead to solutions of mediocre accuracy.

If you use SCS, you should change its termination criteria to be more strict. You should also say what version you used.

Reply:
Would re-running these tests with `MOSEK` with the default solver settings be fair?

Reply:
Re-ran all of these tests using `MOSEK`, and updated the docstring with the version I used. Two of those tests were already run using `MOSEK` XD, I just failed to document that.